cypress/integration/host/host_events.ts

```diff
@@ -1,8 +1,14 @@
 import { clickOnPageSizeBtnAndAssertURLandTableSize } from "../../utils";
 
-const pathWithEvents = `/host/i-0f81a2d39744003dd`;
+const pathWithEvents = "/host/i-0f81a2d39744003dd";
 
 describe("Host events", () => {
+  beforeEach(() => {
+    cy.session("log in and set pagination", () => {
+      cy.login();
+      window.localStorage.setItem("recentPageSize", "20");
+    });
+  });
```

Comment:
Based on my testing, I think this would also work and would avoid from having to login again. Not sure which is the better approach, but just suggesting an alternative :]

```suggestion
  beforeEach(() => {
     cy.window().then((win) => {
         win.localStorage.setItem("recentPageSize", "20");
    });
  });
```

Reply:
Bumping for consistency!